samples/sensor/bme680/prj.conf

```diff
@@ -0,0 +1,28 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_BME680=y
+
+## Additional configuration options
+# Alternate I2C address. Default is 0x76
+# Set the SDO pin to GND for 0x76 and VDDIO for 0x77
+#CONFIG_BME680_I2C_ADDRESS=0x77
+
+# Alternate I2C device. Default is I2C_0
+#CONFIG_BME680_I2C_NAME=I2C_1
+
+# Alternate Temperature, Pressure and Humidity oversampling. Defaults are below
+# Options are 1X, 2X, 4X, 8X and 16X
+#CONFIG_BME680_TEMP_OVER_2X=y
+#CONFIG_BME680_PRESS_OVER_16X=y
+#CONFIG_BME680_HUMIDITY_OVER_1X=y
```

Comment:
Don't add commented out code like this. Either enable the configs or remove the lines.

Reply:
This still needs to be addressed